index.html

```diff
@@ -1,14 +1,339 @@
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div>
+        <div class="chef">
+          <div class="image-container">
+            <img src="./Assets/chef1.jpg" alt="Chef" />
+          </div>
+          <div class="chef-info">
+            <p class="chef-name">Najat kaanach</p>
+            <p class="chef-title">
+              Moroccan Chef Najat Kaanache Wins “Chef Pioneer” Award.
+            </p>
+            <hr />
+            <p class="more-info">
+              Najat Kaanache is the host of AMC Networks' cooking series "Cocina
+              Marroquí" which airs on Canal Cocina in Spain.
+            </p>
+          </div>
+        </div> -->
```

Comment:
Kindly take away every commented piece of code from your project as it makes your codebase look messy and less professional👍.